code/controllers/configuration.dm

```diff
@@ -386,6 +389,12 @@
 						global.cross_allowed = 1
 				if("cross_comms_name")
 					cross_name = value
+				if("panic_server_name")
+					panic_server_name = value
+				if("panic_server_address")
+					global.panic_address = value

```

Comment:
The reasons we left the linked server out, is because other wise admins could change the address and record the key.

for just the bouncer, there is no security list because it doesn't send the coms key to the target.

also, with the new vv, the proper way to do that is to override vv_edit_var for read only to block the change, and override vv_get_var for secret shit to block reading.